frontend/packages/client/src/api/leaderboard.js

```diff
@@ -0,0 +1,17 @@
+import { checkResponse } from '../utils';
+import { COMMUNITIES_URL } from './constants';
```

Comment:
looks like there's a double `/` . Here and then on line 6 it's adding another `/` Causing a fetch error.

Reply:
Fixed.